README.md

```diff
@@ -1,77 +1,58 @@
-<!-- This should be the location of the title of the repository, normally the short name -->
-# repo-template
+# compliance-trestle-arc42-demo
 
-<!-- Build Status, is a great thing to have at the top of your repository, it shows that you take your CI/CD as first class citizens -->
-<!-- [![Build Status](https://travis-ci.org/jjasghar/ibm-cloud-cli.svg?branch=master)](https://travis-ci.org/jjasghar/ibm-cloud-cli) -->
+This demonstration is designed to show how trestle author can be used on architectural template to enforce consistency within an organisation.
```

Comment:
i would put back tics on 'trestle author'.  it sounds like you are talking about a person as a trestle author - rather than the subcommand.

Reply:
And I would describe trestle and trestle author briefly.

Reply:
And briefly elaborate on "enforce consistency." - i.e. make sure additions to the content adhere to the template structure.  Just a few words can make this stuff much more clear.

Reply:
Good point.